src/components/Categories/Categories.js

```diff
@@ -1,9 +1,14 @@
 import React from 'react';
 import './Categories.css';
+import { useDispatch } from 'react-redux/es/exports';
+import { checkStatus } from '../../redux/categories/categories';
 
-const Categories = () => (
-  <>
-    <button className="Check-status" type="button">Check status</button>
-  </>
-);
+const Categories = () => {
+  const dispatch = useDispatch();
+  return (
+    <>
+      <button className="Check-status" onClick={() => dispatch(checkStatus())} type="button">Check status</button>
```

Comment:
- [ ] Please make the dispatch for the `check status` when the button is clicked to display the under construction, right now the `Onclick` is empty.